shorten or redirect the reachout section (#175)
CaptainSifff/paper_teaching-learning-RSE: competencies.md

  knowledge of RC, USERS, and also NEW, in order to effectively interact with
  domain scientists. Today, an example is a presence on social media. The other
  TEAM-related aspect is the embedding of RSE graduates into the community of
  RSEs, sharing the same set of values and competencies. We envision our RSE graduates to be a part of a strong network of other

>   RSEs, tool-related communities and the classical domain communities, making them more effective at supporting research.

Reviewer: ```suggestion
  RSEs, tool-related communities, and the classical domain communities, making them more effective at supporting research.
```

social skill-set focused specialisations
-->

# Connecting RSEs {#sec:reachout}
RSEs gather around a uniform set of values and competencies and thereby 
naturally form networks.

> For an RSE it is important to be a part of a network with other RSEs, irrespective of the career level in order to be more

Reviewer: ```suggestion
For an RSE, it is important to be part of a network with other RSEs, regardless of career level, in order to be more
```

effective at supporting research.

> A perfect first step for forming this network is topic-related conferences, workshops or meet-ups.

Reviewer: ```suggestion
A perfect first step for forming this network is topic-related conferences, workshops, or meet-ups.
```

Reviewer 2: Also not sure if it shouldn't be 'are' instead of 'is' as we are then starting with a list. 
At least that's my gut feeling, but it might be wrong.
@jcohen02 any thoughts on this?

Reviewer 3: I think "is" is correct here @jpthiele since the subject the "first step" is singular.

Beyond that, there is the broader RSE community organised at the local and regional level with chapters or local/regional communities, at the national level with societies and the international RSE society.
This available layered network can greatly benefit the RSE in finding help with issues outside of their own comfort zone
and provides a welcoming, social safety net providing a home for the RSE. Since we feel providing aspiring RSEs with this net

> is of utmost importance we envision compulsory events introducing that to young RSEs.

Reviewer: ```suggestion
is of utmost importance, we envision compulsory events introducing that to young RSEs.
```

> is of utmost importance we envision compulsory events introducing that to young RSEs.

Reviewer: I am not sure I understand or agree with the "we envision compulsory events introducing that to young RSEs":

1. Nothing else in this paper is described as "compulsory"
2. Who likes being told "you have to join this event"? I think motivation is something one should definitely bring, but compulsory events are strange.
3. I would see this similarly to any professional joining conferences and similar events common in their profession. Continuous learning (e.g., via networking) is a trait of every professional that wants to keep doing a good job.

Author: The idea here was more that already in their masters courses available RSE networks are introduced. I don't know about your studies, but I think most people are just left at the end of their master's degree, when they are basically thrown out of university. The idea here is to emphasize the social aspect that is also in the communication skills and to integrate graduating RSEs as soon as possible into the existing networks.

Reviewer: I studied CSE and we were also just left alone to continue. The same I think holds for every studies, but I think the focus here is the profession, not the studies.

I still think that it is kind of a common problem in almost every profession, and that this section does not really add value.

What would make more sense would be to shift the focus to lifelong learning: After I graduate from M.Sc. RSE and I also specialize into something, and I start working in team projects, how can I keep learning?

Even just renaming this section to `Lifelong learning` would improve how this section fits to the rest of the paper.

Reviewer 4: "compulsory events" does sound very strong. Maybe tune it down a little and say

"we envision that young RSEs are alerted to these events"

Reviewer 4: I am happy with @MakisH suggestion to shorten this to a few sentences and move them to the intro. I think what is described here holds for all graduates of any degree.

Author: OK, since @MakisH succesfully shortened the intro to the length it now has, I will not add onto that.
The networking aspect is in 4.4, Tasks and Responsibilities already. Therefore it will be integrated there, mostly

need to have in order to effectively communicate our values.
We will put forward ideas on how to build up bachelor's and master's programmes,
of which a glimpse can already be found in this paper's appendix.
We will show how we intend to provide the necessary continuous education for RSEs after graduation

> and we will connect that with the integration of RSEs into a mesh of community networks aimed at supporting research

Reviewer: ```suggestion
and we will connect that with the integration of RSEs into a mesh of community networks aimed at supporting research,
```
